routing/edge_estimator.cpp

```diff
@@ -25,8 +25,14 @@ inline double TimeBetweenSec(m2::PointD const & from, m2::PointD const & to, dou
 {
   ASSERT_GREATER(speedMPS, 0.0, ());
 
+  // Current time estimation are too optimistic.
+  // Need more accurate tuning: traffic lights, traffic jams, road models and so on.
+  // Add some penalty to make estimation of a more realistic.
+  // TODO: make accurate tuning, remove penalty.
+  double constexpr kTimePenalty = 1.8;
```

Comment:
in future would be good to add turns costs to the estimation, like:
right turn - +5 sec
left turn - + 15 sec
go straight - 0 sec
u-turn - 10 sec
(with respecting to road side - left or right)